test/jest/integration-capture.spec.ts

```diff
@@ -122,4 +201,35 @@ describe('Integration Capture', () => {
             expect(customFlags).toEqual({});
         });
     });
+
+    describe('#facebookClickIdProcessor', () => {
+        it('returns a formatted clickId if it is passed in as a full click id', () => {
+            const fullClickId = 'fb.1.1554763741205.AbCdEfGhIjKlMnOpQrStUvWxYz1234567890';
+
+            const expectedClickId = 'AbCdEfGhIjKlMnOpQrStUvWxYz1234567890';
+
+            expect(facebookClickIdProcessor(fullClickId)).toEqual(expectedClickId);
+        });
+
+        it('returns a formatted clickId if it is passed in as a partial click id', () => {
+            const partialClickId = 'AbCdEfGhIjKlMnOpQrStUvWxYz1234567890';
+
+            const expectedClickId = 'AbCdEfGhIjKlMnOpQrStUvWxYz1234567890';
+
+            expect(facebookClickIdProcessor(partialClickId)).toEqual(expectedClickId);
+        });
+
+        it('returns an empty string if the clickId is not valid', () => {
+            const expectedClickId = '';
+
+            expect(facebookClickIdProcessor(null)).toEqual(expectedClickId);
+            expect(facebookClickIdProcessor(undefined)).toEqual(expectedClickId);
+            expect(facebookClickIdProcessor('')).toEqual(expectedClickId);
+
+            // @ts-ignore
+            expect(facebookClickIdProcessor(NaN)).toEqual(expectedClickId);
+            // @ts-ignore
+            expect(facebookClickIdProcessor(0)).toEqual(expectedClickId);
```

Comment:
instead of the // @ts-ignore comment can you do `NaN as ProcessorFunction) and `0 as ProcessorFunction`?